addons/reloadlaunchers/stringtable.xml

```diff
@@ -18,6 +18,12 @@
             <Chinese>裝載發射器</Chinese>
             <Turkish>Fırlatıcıyı Yükle</Turkish>
         </Key>
+        <Key ID="STR_ACE_ReloadLaunchers_LoadingStarted">
+            <English>is loading your launcher.</English>
+        </Key>
+        <Key ID="STR_ACE_ReloadLaunchers_LoadingAborted">
+            <English>stopped loading your launcher.</English>
+        </Key>
```

Comment:
This is not going to work for some languages, even though in this case I think it should for every language Arma supports. Not sure if we do this anywhere else, but probably want something like `%1 is loading your launcher`, and format on that.

Reply:
Yeah I can format it like that instead.